ci/builders/mac_unopt.json

```diff
@@ -13,7 +13,8 @@
                 "debug",
                 "--unoptimized",
                 "--no-lto",
-                "--prebuilt-dart-sdk"
+                "--prebuilt-dart-sdk",
+                "--impeller-enable-3d"
```

Comment:
Will this unopt build get collected into artifacts that are shipped to Flutter users in any form? The `--impeller-enable-3d` flag also switches on the "experimental" version of `dart:ui`, which allows for appending scene nodes that we should continue not exposing to Flutter users.

Reply:
we don't ship unopt artifacts